spec.html

```diff
@@ -35671,6 +36081,94 @@ <h1>Runtime Semantics: CompileToCharSet</h1>
           <p>A `-` character can be treated literally or it can denote a range. It is treated literally if it is the first or last character of |ClassRanges|, the beginning or end limit of a range specification, or immediately follows a range specification.</p>
         </emu-note>
 
+        <!-- ClassUnion -->
```

Comment:
This whole chunk of new stuff lands between `NonemptyClassRangesNoDash` and `ClassAtom`, which doesn't really make sense. Putting it after `ClassEscape` (as in the grammar) would be better.

Reply:
Done. I moved it all the way after the children of ClassEscape. I also moved ClassCharacter after ClassStrings, as you suggested in the grammar. 73d434a